### PR TITLE
Add theme styles

### PR DESCRIPTION
**Pull request by delbaoliveira**
- [x] Add theme styles
- [x] Add font stack
- [x] Replace Font Awesome link and add icon component

### CODE REVIEW

File: components/~common/Dropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <div key={option.value}>
+              <div className="label" onClick={() => chooseOption(option)}>
+                {option.label}
+              </div>
+            </div>
+          ))}
+        </Options>
+      ) : null}
+    </div>
+  );
```
Comment (SorinC6) [inline components/~common/Dropdown.jsx]: Nice work on building this dropdown Component from scratch 🏅

File: components/~common/Skip.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+function Skip() {
+  return (
+    <S>
+      <p>Skip</p>
+      <div className="dash" />
```
Comment (SorinC6) [inline components/~common/Skip.jsx]: maybe for the future `Skip` show be injected as a prop in the component for more reusability

Comment (delbaoliveira) [reply components/~common/Skip.jsx]: Yeah, I was thinking the same, or maybe we just wrap it in a link component? 🤔

Comment (SorinC6) [reply components/~common/Skip.jsx]: yes.. it will work both ways I think 😃 , we'll see about this after the app will get bigger 👍

**Pull request merged**